docs/02-developers/04-quickstart/remix.md

```diff
@@ -0,0 +1,180 @@
+
+Return to File explorer and right click on the contract you want to deploy, and click on **Compile Storage.sol**. 
+
+![Compile Solidity Contract](/img/developers/quickstart/5-remix.png)
+
+If successful, you will see a checkmark (highlighted as selected), this is necessary so that in the next step the compiled contracts can be detected for deployment.
+
+![Compile Solidity Contract Button](/img/developers/quickstart/5a-remix.png)
+
+## Deploying the Contract
+
+In the left menu, click on **Deploy and run transactions**. Ensure the Environment and Account are correctly set as explained in the first step. Under Contract, ensure your contract is selected. Click on Deploy and it will ask you to sign a transaction in Metamask.
+
+> If the contract had any constructor arguments, those inputs would appear next to the Deploy button for you to fill them.
+
+![Deploy and Run Transactions](/img/developers/quickstart/6-remix.png)
+
+Confirm the transaction in MetaMask:
+
+![Confirm transaction deployment in MetaMask](/img/developers/quickstart/7-remix.png)
+
+If deployment is successful, it will appear under **Deployed/Unpinned Contracts** section
+
+![View deployed/unpinned contract](/img/developers/quickstart/8-remix.png)
+
+## Interacting with the Contract
+
+As shown in the previous step after deploying the contract, you can expand the desired contract and interact with it, calling the available functions by entering the required arguments, if any. Once entered the arguments, click on the method name and, in the event that it’s a writing method, Metamask will prompt you to sign the transaction. If the method is read only, it will show the return value under the method name.
+
+## Verifying the Contract on Rootstock Explorer
+
+Go back to Remix file explorer and go to `contracts -> artifacts -> build-info`. In this folder there will be a json file containing the information of the compilation process. You’ll need this in order to verify the contract on the [Rootstock Testnet Explorer](https://explorer.testnet.rootstock.io/). Copy the whole content of the input attribute (just the value; what’s inside the curly brackets) and paste it in a separate json file in your computer or code editor.
+
+![deployed contract artifact](/img/developers/quickstart/9-remix.png)
+
+Copy the contract's address from the Remix deployment output, you can find the contract address under the **Deployed/Unpinned** Contracts.
+
+![Copy contract address](/img/developers/quickstart/10-remix.png)
+
+Visit the [Rootstock Testnet Explorer](https://explorer.testnet.rootstock.io/) and paste the Contract Address in the search field.
+
+![Paste Contract Address - Rootstock Testnet Explorer](/img/developers/quickstart/11-remix.png)
+
+Click on the Code tab and click the button to **Verify Contract**.
+
+![View Contract Address - Rootstock Testnet Explorer](/img/developers/quickstart/12-remix.png)
+
+Select **Standard JSON Input** as the verification method and fill the form:
```

Comment:
Before this section, maybe a preview of the Verifier Page section would be nice. And the user would better understand the explanations below. Something like this:
<img width="1667" alt="image" src="https://github.com/user-attachments/assets/b9213d7e-63bf-4ddb-b1e1-dc0dfe6b201b">